src/System.ServiceModel.Duplex/pkg/System.ServiceModel.Duplex.pkgproj

```diff
@@ -6,7 +6,7 @@
       <SupportedFramework>net45;win8;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Simply don't understand what the supportedFramework entries is supposed to be updated with. We retargeted the ref assembly from netstandard1.1 to netstandard2.0, adding netstandard1.1 to the supportedFramework entries throws an error.